### PR TITLE
Reinstate the wpt lint checking.

### PR DESCRIPTION
**Pull request by Ms2ger**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/13999)

### CODE REVIEW

**Comment by jdm** [comment]
`./mach test-tidy --all` is unhappy.

**Comment by Ms2ger** [comment]
Fixed.

File: python/tidy/servo_tidy/tidy.py [hunk 1]
```diff
@@ -782,8 +782,8 @@ def collect_errors_for_files(files_to_check, checking_functions, line_checking_f
                     yield (filename,) + error
 
 
-def get_wpt_files(only_changed_files, progress):
-    wpt_dir = os.path.join(".", "tests", "wpt" + os.sep)
+def get_wpt_files(suite, only_changed_files, progress):
+    wpt_dir = os.path.join(".", "tests", "wpt", suite, "")
```
Comment (Ms2ger) [inline python/tidy/servo_tidy/tidy.py]: I'd prefer not doing that in this PR.

**Pull request merged**